con-master/src/main/java/ba/codecentric/master/controller/HotelController.java

```diff
@@ -19,7 +23,15 @@ public HotelController(HotelService hotelService) {
     }
 
     @RequestMapping(value = "/hotel/save", method = RequestMethod.POST)
-    public void saveHotel(@RequestBody Hotel hotel) {
-        hotelService.saveHotel(hotel);
+    public ResponseEntity<Hotel> saveHotel(@RequestBody Hotel hotel) {
+        return new ResponseEntity<Hotel>(hotelService.saveHotel(hotel), HttpStatus.OK);
+
+    }
+
+    @RequestMapping(value = "/hotel/get", method = RequestMethod.GET)
```

Comment:
Use RESTful approach. "hotel" or "hotels" with HTTP GET method ought to be enough.